src/PetConstructor.js

```diff
@@ -0,0 +1,82 @@
+/* eslint-disable func-names */
+const petConfig = {
```

Comment:
I think all the comments made in `PetClass.js` still stand here, in the spirit of DRY I wont re-add them all again for you here!